scrabble.js

```diff
@@ -1,8 +1,204 @@
-var Scrabble = function() {};
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
-};
+  var prompt = require('prompt');
+  var request = require('request');
 
+// SCRABBLE
+  Scrabble = function() {
+    this._points = {
+      A: 1,
```

Comment:
It seems reasonable to declare this as a global constant. If you have lots of Scrabble objects instantiated, each one will have its own this._points allocated in memory. A global constant would be more memory efficient since there would only be one no matter how many Scrabble games there are.